spark/common/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/raster/RasterBandAccessors.scala

```diff
@@ -45,6 +52,34 @@ case class RS_SummaryStats(inputExpressions: Seq[Expression]) extends InferredEx
   }
 }
 
+case class RS_Band(inputExpressions: Seq[Expression]) extends Expression with SerdeAware with CodegenFallback {
```

Comment:
Why not use the inferredexpression?

Reply:
That makes it so it can't find the RS_Band function, have to use Expression.

Reply:
Why is this happening? We need to use inferredExpression whenever we can.

Reply:
Oh didn't know that, Will look into it. I referred to a function that use Expression instead of InferredExpression, ST_Collect being one. Will ping you when I know more.